libraries/anaconda3-buildtime/context/build

```diff
@@ -4,6 +4,11 @@ set -e
 
 ROOTDIR=/opt/algorithm
 
-. /opt/conda/etc/profile.d/conda.sh
 
-conda env update --prefix $ANACONDA_ENV --file=$ROOTDIR/requirements.yml
+. /opt/conda/etc/profile.d/conda.sh
+if [ -z "$1" ]; then
+    conda env update --prefix "$ANACONDA_ENV" --file $ROOTDIR/requirements.yml;
```

Comment:
nit: why are there semi-colons at the end of these lines?
Also, I don't think you need the quotes around `$ANACONDA_ENV`

Reply:
I threw this though a shell linter and it recommended "'s to remove ambiguity, I'm down with removing em